src/scalap/scala/tools/scalap/scalax/rules/scalasig/ScalaSig.scala

```diff
@@ -246,7 +246,9 @@ object ScalaSigEntryParsers extends RulesWithState with MemoisableRules {
       32 -~ longValue ^^ (java.lang.Double.longBitsToDouble),
       33 -~ nameRef,
       34 -^ null,
-      35 -~ typeRef)
+      35 -~ typeRef,
+      36 -~ symbolRef
```

Comment:
Review by @lrytz, this is 2009 [calling](https://github.com/scala/scala/commit/c762ae353b9ff8beec6ed9516b70134e1f04534d#diff-de53eb5601161b428c5d8a57b0a81737R57) :)

Reply:
> more annotations cleanup.

Great commit message for a +159 -194 commit 😂